es.hjson

```diff
@@ -850,12 +850,12 @@
 
   "dox.gendersArray": [
     "chico"
-    "niña"
+    "chica"
     "e-chico"
-    "e-niña"
+    "e-chica"
     "thot"
     "puta"
-    "fuckboy"
+    "jodido chico / puto"

```

Comment:
Does "fuckboy" have a better translation? I don't think "jodido chico" is good enough here.

You can also completely swap it out for a different insult.